src/main/java/seedu/address/storage/JsonAdaptedGroup.java

```diff
@@ -22,21 +22,27 @@
     public static final String MISSING_FIELD_MESSAGE_FORMAT = "Group's %s field is missing!";
 
     private final String groupName;
+    private final List<JsonAdaptedPerson> persons = new ArrayList<>();
     private final List<JsonAdaptedTask> tasks = new ArrayList<>();
 
     /**
      * Constructs a {@code JsonAdaptedGroup} with the given group details.
      */
     @JsonCreator
-    public JsonAdaptedGroup(@JsonProperty("groupName") String groupName) {
+    public JsonAdaptedGroup(@JsonProperty("groupName") String groupName,
+                            @JsonProperty("persons") List<JsonAdaptedPerson> persons) {
```

Comment:
I think this is being stored wrongly. Reason is that a list of persons (i.e. student contact) is contained in the `Group`. But a student/student list is not an attribute of a `Group`.  The only attribute of a group is the `GROUP_NAME` (can see from the command `addgroup g/GROUP_NAME`). 

I think this won't affect the implementation at v1.2, but the storing of the json file would be different. 

We can remove `@JsonProperty("persons") List<JsonAdaptedPerson> persons` so it would just be `public JsonAdaptedGroup(@JsonProperty("groupName") String groupName)` as per original.